ecms_base/themes/custom/ecms/ecms.info.yml

```diff
@@ -0,0 +1,28 @@
+name: Rhode Island eCMS Theme
+description: "Drupal installation profile for the State of Rhode Island's eCMS system"
```

Comment:
Nit pick, but I think this description should change to mention it is a custom Drupal theme for the eCMS system.

Reply:
Ah good catch! Fixed that